### DISCUSSION TITLE
PieChart: Support thresholds and gradient color schemes

### DISCUSSION THREAD

**Discussion by jaakoob** [grafana/grafana#38258] [Dashboards, Panels & Visualizations]
**What happened**:

Thresholds are not applied in pie-chart v2 panel
![2021-04-08 15-50-07](https://user-images.githubusercontent.com/32491165/114038426-2773c100-9882-11eb-8b51-edecf0c07b53.png)

**What you expected to happen**:

The color of inividual sections changes as matching thresholds are defined.

**How to reproduce it (as minimally and precisely as possible)**:

Add a pie chart v2 to a panel with a random walk datasource. The section in the pie chart does not turn green or red (predefined thresholds, green: base, red: >= 80)

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.5.3
- Data source type & version: Random Walk/ Elasticsearch
- OS Grafana is installed on: Centos 8.3
- User OS & Browser: Fedora 33, Firefox 87.0
- Grafana plugins: Pie chart v2 (beta)
- Others:

**Comment by oscarkilhed** [comment]
Currently not sure how to implement thresholds in the pie chart. Having all the slices matching a threshold turn the same color isn't very useful. The options are currently in line with stuff like gauges and stat panels, but the panel does not handle colors in the same way.

Maybe this option should be cut?

**Comment by torkelo** [comment]
It would work but we have not enabled the color scheme "From thresholds" for pie chart yet. We could do that, but not sure thresholds make sense for a pie chart?

**Comment by oscarkilhed** [comment]
> but not sure thresholds make sense for a pie chart?

This was my conclusion as well after looking a bit deeper into this. Made this PR #33375

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
